dali/operators/decoder/image_decoder.cc

```diff
@@ -106,74 +141,109 @@ threads given to the DALI pipeline and should be found empirically.)code",
-slice (x0, x1, x2, ...), and `shape` containing either normalized or absolute coordinates
-(depending on the value of `normalized_shape`) for the dimensions of the slice
-(s0, s1, s2, ...). Both `anchor` and `shape` coordinates must be within the interval
-[0.0, 1.0] for normalized coordinates, or within the image shape for absolute
-coordinates. Both `anchor` and `shape` inputs will provide as many dimensions as specified
-with arguments `axis_names` or `axes`.
-
-By default `ImageDecoderSlice` operator uses normalized coordinates and `WH` order for the slice
-arguments.
-
-When possible, it will make use of region-of-interest decoding APIs (e.g. *libjpeg-turbo*, *nvJPEG*)
-thus optimizing decoding time and memory usage. When not supported, it will decode the whole image
-and then crop the selected ROI.
-
-Note: ROI decoding is currently not compatible with hardware based decoding.
-Using *ImageDecoderSlice* will automatically disable hardware accelerated decoding.
-To make use of the hardware decoder, use *ImageDecoder* and *Slice* operators instead.
-
-The output of the decoder is in *HWC* layout.
-
-Supported formats: JPG, BMP, PNG, TIFF, PNM, PPM, PGM, PBM.)code")
+  .DocStr(R"code(Decodes images and extracts an externally provided ROI that are specified by an
+anchor and a shape of the ROI.
+
+Inputs must be supplied as tensors in the following order:
+
+* ``data`` that contains the input data.
+* An ``anchor`` that contains normalized or absolute coordinates, depending on the
+  ``normalized_anchor`` value, for the starting point of the slice (x0, x1, x2, and so on),
+* ``shape`` that contains normalized or absolute coordinates, depending on the
+  ``normalized_shape`` value, for the dimensions of the slice (s0, s1, s2, and so on).
+
+The anchor and shape coordinates must be within the interval [0.0, 1.0] for normalized
+coordinates or within the image shape for the absolute coordinates. The ``anchor`` and ``shape``
+inputs will provide as many dimensions as were specified with arguments ``axis_names`` or ``axes``.
+
+By default, the :meth:`nvidia.dali.ops.ImageDecoderSlice` operator uses normalized coordinates
+and "WH" order for the slice arguments.
+
+When possible, it will make use the ROI decoding APIs (for example, *libjpeg-turbo* and *nvJPEG*)
+that optimizes decoding time and memory usage. If the decoding APIs are not supported,
+the operator will decode the entire image and crop the selected ROI.
+
+.. note::
+  ROI decoding is currently not compatible with hardware-based decoding. Using
+  :meth:`nvidia.dali.ops.ImageDecoderSlice` automatically disables hardware accelerated decoding.
+  To use the hardware decoder, use the :meth:`nvidia.dali.ops.ImageDecoder` and
+  :meth:`nvidia.dali.ops.Slice` operators instead.
+
+The output of the decoder is in the *HWC* layout.
+
+The followingformats are supported:
+
+* JPG
+* BMP
+* PNG
+* TIFF
+* PNM
+* PPM
+* PGM
+* PBM)code")
```

Comment:
Again, same remark about format. They're not important.

Reply:
Done